better-font-awesome.php

```diff
@@ -513,18 +546,52 @@ public function text_callback( $args ) {
-                     <i class="icon-coffee fa fa-coffee"></i> <code>[icon name="coffee"]</code> or <code>&lt;i class="fa-coffee"&gt;&lt;/i&gt;</code><br /><br />
-                     <i class="icon-coffee fa fa-coffee icon-2x fa-2x"></i> <code>[icon name="coffee" class="fa-2x"]</code> or <code>&lt;i class="fa-coffee fa-2x"&gt;&lt;/i&gt;</code><br /><br />
-                     <i class="icon-coffee fa fa-coffee icon-2x fa-2x icon-rotate-90 fa-rotate-90"></i> <code>[icon name="coffee" class="fa-2x fa-rotate-90"]</code> or <code>&lt;i class="fa-coffee fa-2x fa-rotate-90"&gt;&lt;/i&gt;</code><br /><br /><br />
-                     <b>Font Awesome version 3.x</b>&nbsp;&nbsp;&nbsp;<small><a href="http://fontawesome.io/3.2.1/examples/">See all available options &raquo;</a></small><br /><br />
-                     <i class="icon-coffee fa fa-coffee"></i> <code>[icon name="coffee"]</code> or <code>&lt;i class="icon-coffee"&gt;&lt;/i&gt;</code><br /><br />
-                     <i class="icon-coffee fa fa-coffee icon-2x fa-2x"></i> <code>[icon name="coffee" class="icon-2x"]</code> or <code>&lt;i class="icon-coffee icon-2x"&gt;&lt;/i&gt;</code><br /><br />
-                     <i class="icon-coffee fa fa-coffee icon-2x fa-2x icon-rotate-90 fa-rotate-90"></i> <code>[icon name="coffee" class="icon-2x icon-rotate-90"]</code> or <code>&lt;i class="icon-coffee icon-2x icon-rotate-90"&gt;&lt;/i&gt;</code>',
-                'better-font-awesome' ) .
-                '</div>';
+        return '<div class="bfa-usage-text">' .
+
+                    '<h3>' . esc_html__( 'Usage', 'better-font-awesome' ) . '</h3>' .
+                    sprintf(
+	                    esc_html__(
+		                    'Font Awesome version 4.x + %s See all available options %s',
+		                    'better-font-awesome'
+	                    ),
+	                    '&nbsp;&nbsp;&nbsp;<small><a href="http://fontawesome.io/examples/">',
+	                    '&raquo;</a></small><br/><br/>'
+                    ) .
+                    sprintf(
+	                    '<i class="icon-%$1s fa fa-%$1s"></i> <code>[icon name="%$1s"]</code> %$2s <code>&lt;i class="fa-%$1s"&gt;&lt;/i&gt;</code><br /><br />' .
+	                    '<i class="icon-%$1s fa fa-%$1s icon-2x fa-2x"></i> <code>[icon name="%$1s" class="fa-2x"]</code> %$2s <code>&lt;i class="fa-%$1s fa-2x"&gt;&lt;/i&gt;</code><br /><br />' .
+	                    '<i class="icon-%$1s fa fa-%$1s icon-2x fa-2x icon-rotate-90 fa-rotate-90"></i> <code>[icon name="%$1s" class="fa-2x fa-rotate-90"]</code> %$2s <code>&lt;i class="fa-%$1s fa-2x fa-rotate-90"&gt;&lt;/i&gt;</code><br /><br /><br />',
+	                    esc_html__(
+		                    'coffee',
+		                    'better-font-awesome'
+	                    ),
+	                    esc_html__(
+		                    'or',
+		                    'better-font-awesome'
+	                    )
+                    ) .
```

Comment:
@dashaluna you want something like the following. I did a quick conversion to html on the first section for you. The best foundation for good code is readability, this method is much easier to read and interpret so much easier to improve and bug fix.

```
public function get_usage_text() {

	ob_start();
	?>
	<div class="bfa-usage-text">
		<h3><?php esc_html_e( 'Usage', 'better-font-awesome' ); ?></h3>
		<b><?php printf( esc_html__( 'Font Awesome version %s' ), '4.x' ); ?></b>

		<small>
			<a href="http://fontawesome.io/examples/"><?php esc_html__( 'See all available options', 'better-font-awesome' ); ?>&raquo;</a>
		</small>

		<br/>

		<?php $v1 = esc_html__( 'coffee', 'better-font-awesome' ); //Are you sure all instances should be translated? looks like classnames etc will get messed up?>
		<?php $v2 = esc_html__( 'or', 'better-font-awesome' ); ?>

		<i class="icon-<?php echo $v1; ?> fa fa-<?php echo $v1; ?>"></i>

		<code>[icon name="<?php echo $v1; ?>"]</code>
		<?php echo $v2; ?>
		<code>
			<i class="fa-<?php echo $v1; ?>"></i>
		</code>

		<br /><br />

		<i class="icon-<?php echo $v1; ?> fa fa-<?php echo $v1; ?> icon-2x fa-2x"></i>

		<code>[icon name="<?php echo $v1; ?>" class="fa-2x"]</code>
		<?php echo $v2; ?>
		<code>
			<i class="fa-<?php echo $v1; ?> fa-2x"></i>
		</code>

		<br /><br />

		<i class="icon-<?php echo $v1; ?> fa fa-<?php echo $v1; ?> icon-2x fa-2x icon-rotate-90 fa-rotate-90"></i>

		<code>[icon name="<?php echo $v1; ?>" class="fa-2x fa-rotate-90"]</code>
		<?php echo $v2; ?>
		<code>
			<i class="fa-<?php echo $v1; ?> fa-2x fa-rotate-90"></i>
		</code>

		<br /><br /><br />
	</div>
	<?php
	return ob_get_clean();
}
```
Should ideally be templating this too rather than having a function responsible for generating so much html content.